CMakeLists.txt

```diff
@@ -816,7 +816,6 @@ endif()
 if(OCOS_USE_CUDA)
   target_include_directories(ortcustomops PUBLIC ${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES})
   target_link_libraries(extensions_shared PUBLIC cudart cublas cufft)
-  set_target_properties(ocos_operators PROPERTIES CUDA_SEPARABLE_COMPILATION ON)
```

Comment:
Do you have some insight about which way is better?

Reply:
The CUDA_SEPARABLE_COMPILATION cause link issues.
Removing it  helper the build pass.